elfi/methods/bo/acquisition.py

```diff
@@ -745,6 +745,9 @@ def evaluate(self, theta_new, t=None):
         w = ((self.phi_int - phi_skew_imp) / 2)
 
         loss_theta_new = 2 * np.sum(self.omegas_int * self.priors_int * w, axis=1)
+        loss_theta_new = np.where(self.prior.pdf(theta_new) == 0,
+                                  np.max(np.abs(loss_theta_new)) * 1e10,
```

Comment:
nice i didn't know `np.where` can be used in substitutions like this ^^ just want to make sure about the value that is used: is there a reason we cannot use `np.inf` or another fixed value like `np.finfo(float).max` here? since that would avoid the extra operations on this line

Reply:
Good suggestion! Committing the change now.